docs/source/testing-and-issue-tracking/Kokkos-Project-Planning.md

```diff
@@ -0,0 +1,210 @@
+available on currently Kokkos supported software stacks to the greatest extent possible.
+
+#### Activities to support this requirement:
+
+- participate in ISO C++ committee meetings
+- monitor requests for ISO C++ features to be provided by Kokkos
+- write proposals for ISO C++ for mature Kokkos features with wide applicability
+- backport relevant future ISO C++ features to standards supported by Kokkos
+
+## Release Planning
+
+Kokkos releases are based on the "catch the train" model - i.e. the primary goal is to have regular releases,
+not a specific feature list for each release.
+
+Major releases happen every three years, minor releases are aimed at every 3-4 months, with additional patch releases as necessary.
+
+The primary difference between a major and a minor release is that deprecated features are only removed at major releases, and
+major releases come with a bump in minimal compiler version requirements, and an updated minimum ISO C++ standard version.
+Other than that, there is no difference in planning and execution of major and minor releases.
+
+The difference of patch releases to minor and major releases is that patch releases  will generally only contain bugfixes, and not new capabilities.
+
+At the beginning of a release cycle the Kokkos Core leadership will determine high priority thrusts for the release cycle.
+Furthermore, each team member will make a list of their personal priorities for the release cycle.
+The priorities are discussed and refined at the Kokkos developer meeting and collected in internal documents.
+
+Issues for each item are assigned to the [Kokkos project plan](https://github.com/orgs/kokkos/projects/1) including team member assignments.
+
+The [Kokkos project plan](https://github.com/orgs/kokkos/projects/1) assigns issues to one of 7 categories:
+
+- *Unassigned:* issues that aren't assigned yet to team members.
+- *Unassigned - Priority:* issues that aren't assigned yet to team members, but are high priority. These should be assigned at the next weekly developer meeting.
+- *To Do:* Issue was assigned to a team member, but is not yet actively worked on.
+- *To Do - Priority:* Issue was assigned to a team member, but is not yet actively worked on. It is expected to be the next item in the queue of the assigned developer. If this item does not transition to *In Progress* by the next developer meeting, reassignment is considered.
+- *In Progress:* Issue is getting worked on.
+- *In Progress - Priority:* Issue is getting worked on. Code reviews for this issue are considered priority, in order to get this merged as soon as possible.
+- *Done:* Issue is addressed via merged pull request, or was closed because of new information which made it obsolete. For merged pull requests it is ensured that a Changelog entry was generated if appropriate, before removing the item from the project plan.
```

Comment:
Note that a lot of (most of?) the items in the planning board are pull requests, not issues. This is particularly confusing for new pull requests that are "Unassigned" by default but probably should rather be "In Progress".

```suggestion
Kokkos releases are based on the "catch the train" model - i.e. the primary goal is to have regular releases,
not a specific feature list for each release.

Major releases happen every three years, minor releases are aimed at every 3-4 months, with additional patch releases as necessary.

The primary difference between a major and a minor release is that deprecated features are only removed at major releases, and
major releases come with a bump in minimal compiler version requirements and an updated minimum ISO C++ standard version.
Other than that, there is no difference in the planning and execution of major and minor releases.

In contrast to major and minor releases, patch releases generally only contain bug fixes and no new capabilities.

At the beginning of a release cycle, the Kokkos Core leadership will determine high-priority thrusts for the release cycle.
Furthermore, each team member will make a list of their personal priorities for the release cycle.
The priorities are discussed and refined at the Kokkos developer meeting and collected in internal documents.

Issues for each item are assigned to the [Kokkos project plan](https://github.com/orgs/kokkos/projects/1) including team member assignments.

The [Kokkos project plan](https://github.com/orgs/kokkos/projects/1) assigns issues to one of 7 categories:

- *Unassigned:* issues that aren't assigned yet to team members.
- *Unassigned - Priority:* issues that aren't assigned yet to team members, but are high priority. These should be assigned at the next weekly developer meeting.
- *To Do:* Issue was assigned to a team member but is not yet actively worked on.
- *To Do - Priority:* Issue was assigned to a team member, but is not yet actively worked on. It is expected to be the next item in the queue of the assigned developer. If this item does not transition to *In Progress* by the next developer meeting, reassignment is considered.
- *In Progress:* Issue is getting worked on.
- *In Progress - Priority:* Issue is getting worked on. Code reviews for this issue are considered a priority, in order to get this resolved as soon as possible.
- *Done:* Issue is addressed via merged pull request, or was closed because of new information which made it obsolete. For merged pull requests it is ensured that a changelog entry was generated, if appropriate, before removing the item from the project plan.
```
Issues don't get "merged".